draft-schinazi-httpbis-unprompted-auth.md

```diff
@@ -111,7 +111,7 @@ generate a 32-byte key which is then used as a nonce.
 The "Unprompted-Authentication" header field allows a user agent to authenticate
 with an origin server. The authentication is scoped to the HTTP request
 associated with this header field. The value of the Unprompted-Authentication
-header field is a token which represents the Unpromted Authentication Scheme;
+header field is a token which represents the HTTP Authentication Scheme;
 see {{schemes}}. This header field supports parameters.
```

Comment:
I think these parameter definitions should be moved into the schemes.  `Unprompted-Authentication` should carry a generic `credentials` value as defined in RFC 9110.

Reply:
Good point, done.